FAQ.md

```diff
@@ -0,0 +1,20 @@
+## FAQ
```

Comment:
I wonder whether this page could be moved to ROADMAP.md, to keep all progress updates in one place (ROADMAP.md already looks out of date).